specs/core/0_beacon-chain.md

```diff
@@ -213,15 +216,13 @@ Unless otherwise indicated, code appearing in `this style` is to be interpreted
 * The `BASE_REWARD_QUOTIENT` parameter dictates the per-epoch reward. It corresponds to ~2.54% annual interest assuming 10 million participating ETH in every epoch.
 * The `INACTIVITY_PENALTY_QUOTIENT` equals `INVERSE_SQRT_E_DROP_TIME**2` where `INVERSE_SQRT_E_DROP_TIME := 2**12 epochs` (~18 days) is the time it takes the inactivity penalty to reduce the balance of non-participating [validators](#dfn-validator) to about `1/sqrt(e) ~= 60.6%`. Indeed, the balance retained by offline [validators](#dfn-validator) after `n` epochs is about `(1-1/INACTIVITY_PENALTY_QUOTIENT)**(n**2/2)` so after `INVERSE_SQRT_E_DROP_TIME` epochs it is roughly `(1-1/INACTIVITY_PENALTY_QUOTIENT)**(INACTIVITY_PENALTY_QUOTIENT/2) ~= 1/sqrt(e)`.
 
-### Status codes
+### Status flags
 
 | Name | Value |
 | - | - |
-| `PENDING_ACTIVATION` | `0` |
-| `ACTIVE` | `1` |
-| `ACTIVE_PENDING_EXIT` | `2` |
-| `EXITED_WITHOUT_PENALTY` | `3` |
-| `EXITED_WITH_PENALTY` | `4` |
+| `INITIATED_EXIT` | `2**0` (= 1) |
+| `WITHDRAWABLE` | `2**1` (= 2) |
```

Comment:
I don't think we need a special constant for this; we have lots of values that we initialize to zero. Though it seems like I forgot to initialize it; fixed now.

Reply:
It's alright if we don't define this initial constant in spec. Just FYI, in Python implementation side, we may make `ValdiatorStatusFlags` as an ` Enum` class, so that we can add type hinting when setting `status_flags`:
```python
class ValidatorStatusFlags(IntEnum):
    INITIAL = 0
    INITIATED_EXIT = 1
    WITHDRAWABLE = 2
```

Reply:
Is enum the appropriate class to use for flags? These are bit positions, not numbers that only get checked for equality (which is the usual use case of enum), so it feels like a sightly different thing.... though I don't care too much.